integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -49,4 +96,14 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
   ): Promise<AxiosResponse<RasaResponse>> {
     return axios.post(`${this.config.serverUrl}${this.config.serverPath}`, { text }, config);
   }
+
+  private mapAlternativeIntents(altIntents: RasaIntent[]) {
+    // remove first element, because its the classified intent
+    altIntents.splice(0, 1);
```

Comment:
If the response from Rasa is persisted in the future (which can and can not happen), this could cause some side effects due to mutating the parameter `altIntents`. In that case, it would probably be better to simply `slice` before filtering.

Reply:
renamed method argument and using slice now